tests/rpmbuild.at

```diff
@@ -2435,6 +2435,32 @@ No hello.debug
 [ignore])
 RPMTEST_CLEANUP
 
+AT_SETUP([manual debuginfo package])
+AT_KEYWORDS([build debuginfo])
+RPMTEST_CHECK([
+RPMDB_INIT
+
+runroot rpmbuild --quiet -bb \
```

Comment:
This needs at least `--define "_prefix /usr/local"` to ensure the paths end up there, otherwise it fails when rpm is built with /usr prefix (ie the normal case)